Mathlib/LinearAlgebra/Matrix/SchurComplement.lean

```diff
@@ -456,14 +456,12 @@ end Det
 
 end CommRing
 
-/-! ### Lemmas about `ℝ` and `ℂ`-/
+/-! ### Lemmas about `ℝ` and `ℂ` and other `StarOrderedRing`s -/
 
 
-section IsROrC
+section StarOrderedRing
 
-open scoped Matrix
-
-variable {𝕜 : Type*} [IsROrC 𝕜]
+variable {𝕜 : Type*} [CommRing 𝕜] [PartialOrder 𝕜] [StarOrderedRing 𝕜]
```

Comment:
Unrelated to this PR: I was surprised to find that `CommRing` is actually necessary here. And that's because it's necessary for, at least, `A⁻¹`. That's a bit of a shame, because all this 2x2 Schur complement stuff (not involving determinants) should work in the non-commutative setting too (IIRC).